.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+version: 2.1
+
+executors:
+  node:
+    parameters:
+      tag:
+        type: string
+    docker:
+      - image: cimg/node:<< parameters.tag >>
+
+commands:
+  install-dependencies:
+    steps:
+      - run:
+          name: "Install project dependencies"
+          command: npm i
+  apt-update:
+    steps:
+      - run:
+          name: "APT update"
+          command: sudo apt-get update
+  install-libscret:
+    steps:
+      - run:
+          name: "Fetch and install libsecret"
+          command: sudo apt-get install -y libsecret-1-dev
+  run-unit-tests:
+    steps:
+      - run:
+          name: "Run unit tests"
+          command: npm run test:ci
+  run-linter:
+    steps:
+      - run:
+          name: "Run linter"
+          command: npm run lint
+
+jobs:
+  pr-validator:
+    resource_class: small
+    environment:
+      NODE_OPTIONS: --max-old-space-size=14336
+      CI: ""
```

Comment:
I was confused about this as well, I believe I pinged you about this, but it might have been lost somewhere. As per [here](https://github.com/twilio/flex-plugin-builder/blob/main/.travis.yml#L15) you are setting it to empty as well, else something in the unit tests is breaking